PMDMMOc/src/com/darkxell/client/mechanics/event/ClientEventProcessor.java

```diff
@@ -55,26 +55,15 @@
 import com.darkxell.common.event.item.ItemLandedEvent;
 import com.darkxell.common.event.item.ItemMovedEvent;
 import com.darkxell.common.event.item.ItemSelectionEvent;
-import com.darkxell.common.event.item.ItemSwappedEvent;
 import com.darkxell.common.event.item.ItemThrownEvent;
 import com.darkxell.common.event.item.MoneyCollectedEvent;
 import com.darkxell.common.event.item.ProjectileThrownEvent;
 import com.darkxell.common.event.move.MoveDiscoveredEvent;
 import com.darkxell.common.event.move.MoveLearnedEvent;
 import com.darkxell.common.event.move.MoveSelectionEvent;
 import com.darkxell.common.event.move.MoveUseEvent;
-import com.darkxell.common.event.pokemon.BlowbackPokemonEvent;
-import com.darkxell.common.event.pokemon.DamageDealtEvent;
+import com.darkxell.common.event.pokemon.*;
```

Comment:
Right, we should talk about when it is a good time to use star import? No action needed.